plugins/woocommerce-admin/client/marketplace/components/product-loader/product-loader.scss

```diff
@@ -0,0 +1,69 @@
+@import '../../stylesheets/_variables.scss';
+
+.woocommerce-marketplace {
+	&__product-loader-cards {
+		display: grid;
+		background: linear-gradient(to right, $gray-0 40%, $gray-10 60%, $gray-0 80%);
+		background-color: $gray-0;
+		background-size: 500% 200%;
+		animation: GradientSlide 4s linear infinite;
+		height: 270px;
+	}
+
+	&__product-loader-divider {
+		background: #fff;
+		width: 24px;
+		height: 270px;
```

Comment:
It works just as well without this, and then if we ever need to adjust the overall height, we can just do it on the container.

```suggestion
```